src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -223,6 +223,9 @@ const PIADetailPage = () => {
       }
 
       if (buttonValue === 'submit') {
+        // here will check if this is the first submit, if so will add submittedAt property
+        // otherwise no action needed
+        if (!pia?.submittedAt) requestBody.submittedAt = new Date();
```

Comment:
This doesn't look like the right place to add the submittedAt value. 

You should do this in backend. Thoughts?

In a common function called updatedSubmittedAt() called from 'patch' and 'create' methods.

`if (!submittedAt && status === 'MPO_REVIEW') {
submittedAt = new Date()
}`